themes/default/data/registry/packages/scaleway.yaml

```diff
@@ -1,14 +1,14 @@
-category: Infrastructure
```

Comment:
Did you want to keep your package categorized as `infrastructure`? If so, you could add a tag `category/infrastructure` to the `Keywords` slice in your [`resources.go`](https://github.com/jaxxstorm/pulumi-scaleway/blob/master/provider/resources.go#L79) file. See the description for `Keywords` in our [docs](https://www.pulumi.com/docs/guides/pulumi-packages/schema/#pulumi-package-schema).